.bazelrc

```diff
@@ -1,3 +1 @@
-common       --config=rules
```

Comment:
Do you know why this existed to begin with?

Why have a `rules` section when we could just put those options under the commands themselves? E.g. instead of `build:rules --disk_cache=.bazel_cache`, we could do `build --disk_cache=.bazel_cache`.

Reply:
These configs existed because way back when Bazel was new the tests and rules had separate config.

Also because of us wanting to have a single place for the disk cache, which I realize I broke by removing this. I'll deal with that.